packages/central-server/src/apiV2/index.js

```diff
@@ -269,6 +270,7 @@ apiV2.get('/entityHierarchy/:recordId?', useRouteHandler(GETEntityHierarchy));
 apiV2.get('/landingPages/:recordId?', useRouteHandler(GETLandingPages));
 apiV2.get('/suggestSurveyCode', catchAsyncErrors(suggestSurveyCode));
 apiV2.get('/tasks/:recordId?', useRouteHandler(GETTasks));
+apiV2.get('/tasks/:parentRecordId/taskComments', useRouteHandler(GETTaskComments));
```

Comment:
The parentRecordId just refers the task_id relation on the comments table, right? Any reason that you didn't just use :taskId here? To me it seems that it would be more obvious what it is referring to if it was taskId

Reply:
Heya, yes I originally called it `taskId` but the central-server has built in logic that handles `parentRecordId` so that's what it needs to be to access that